index.html

```diff
@@ -230,9 +230,9 @@ <h3>Skills</h3>
       </p>
 
       <form class="form-page">
-        <input class="fullname" type="text" id="name" name="name" placeholder="Full name">
+        <input class="fullname" type="text" id="name" name="name" placeholder="Tofcookies">
```

Comment:
- [ ] For all input fields for the form, add the required attribute so that a form is not submitted with missing fields